src/shared/codelens/localLambdaRunner.ts

```diff
@@ -469,12 +440,10 @@ export const invokeLambdaFunction = async (params: {
     samTaskInvoker: SamCliTaskInvoker,
     telemetryService: TelemetryService,
     runtime: string,
-}): Promise<void> => {
-    if (params.isDebug && !params.debugConfig) {
-        const err = new Error('Started debug run without a debugConfig')
-        params.channelLogger.logger.error(err)
-        throw err
-    }
+    debuggerPath?: string
+}
+
+export async function invokeLambdaFunction(params: InvokeLambdaFunctionArguments): Promise<void> {
```

Comment:
[Line 451](https://github.com/aws/aws-toolkit-vscode/pull/513/files#diff-10aae715ff27602da9082470cd015003R451) has a debug logger that should probably have the new params added to it.
(I don't know why I can't add a comment on the line directly)